kms-message/src/kms_b64.c

```diff
@@ -287,8 +285,8 @@ kms_message_b64_initialize_rmap (void)
    }
 
    /* Fill reverse mapping for base64 chars */
-   for (i = 0; Base64[i] != '\0'; ++i)
-      b64rmap[(uint8_t) Base64[i]] = i;
+   for (j = 0; Base64[j] != '\0'; ++j)
+      b64rmap[(uint8_t) Base64[j]] = j;
```

Comment:
```suggestion
   for (i = 0; Base64[i] != '\0'; ++i)
      b64rmap[(uint8_t) Base64[i]] = (uint8_t) i;
```
Suggest simple cast from `int` to `uint8_t`.